### PR TITLE
View/custom view

### PR DESCRIPTION
**Pull request by MadayAlcala**
Hi guys, in this branch attached the portable VLC, please check it out.

### CODE REVIEW

File: src/main/java/com/fundation/logic/view/FooterPanel.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.fundation.logic.view;
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/FooterPanel.java:10]: Please add copyright tag.

File: src/main/java/com/fundation/logic/view/FooterPanel.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+import javax.swing.JPanel;
+import java.awt.Color;
+import java.awt.Label;
+
+public class FooterPanel extends JPanel {
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/FooterPanel.java:22]: Please add documentation

File: src/main/java/com/fundation/logic/view/HeaderPanel.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
+package com.fundation.logic.view;
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/HeaderPanel.java:10]: Please add copyright and documentation.

File: src/main/java/com/fundation/logic/view/PopUpMessage.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
+package com.fundation.logic.view;
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/PopUpMessage.java:10]: Please add copyright and documentation.

File: src/main/java/com/fundation/logic/view/PopUpMessage.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+public class PopUpMessage {
+    /**
+     * Shows a popup message
+     * @param messageTittle - Tittle of the message
+     * @param messageText - Text of the message
+     */
+    public static void showPopupMessage(String messageTittle, String messageText) {
+        JOptionPane.showMessageDialog(null, messageText,
+                messageTittle, JOptionPane.WARNING_MESSAGE);
+//        message.setBounds(50, 200, 350, 150);
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/PopUpMessage.java]: Commented code

File: src/main/java/com/fundation/logic/view/loadSaveCriteria/LoadSavePanel.java [hunk 6]
```diff
@@ -9,15 +9,15 @@
  */
 package com.fundation.logic.view.loadSaveCriteria;
 
-import com.fundation.logic.view.customElements.CustomButton;
 import com.fundation.logic.view.customElements.CustomPanelSecond;
 import com.fundation.logic.view.customElements.CustomTable;
 
-import javax.swing.JTable;
+import javax.swing.*;
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/loadSaveCriteria/LoadSavePanel.java]: please use specific libraries

File: src/main/java/com/fundation/logic/view/HeaderPanel.java [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+import javax.swing.ImageIcon;
+import javax.swing.JLabel;
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.Insets;
+
+public class HeaderPanel extends CustomPanelSecond {
+    public HeaderPanel(){
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/HeaderPanel.java]: space after HeaderPanel() {

File: src/main/java/com/fundation/logic/view/customElements/CustomJFrame.java [hunk 8]
```diff
@@ -9,11 +9,8 @@
  */
 package com.fundation.logic.view.customElements;
 
-import javax.swing.JFrame;
-import javax.swing.JLabel;
-import javax.swing.BorderFactory;
+import javax.swing.*;
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/customElements/CustomJFrame.java]: Use only needed classes.

File: src/main/java/com/fundation/logic/view/customElements/CustomTable.java [hunk 9]
```diff
@@ -26,12 +23,11 @@
...
 
-        Color tableColor = new Color(43,43,43);
+        Color tableColor = new Color(60,63,65);
         setBackground(tableColor);
-        setForeground(Color.WHITE);
+        setFont(new Font("Calibri", 2, 14));
+        setForeground(new Color(237, 234, 229));
         getTableHeader().setBackground(Color.cyan);
-        setGridColor(Color.BLACK);
-
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/customElements/CustomTable.java:34]: unnecessary white lines

File: src/main/java/com/fundation/logic/view/FooterPanel.java (hunk 1 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/FooterPanel.java:10]: Solved.

File: src/main/java/com/fundation/logic/view/HeaderPanel.java (hunk 3 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/HeaderPanel.java:10]: Solved.

File: src/main/java/com/fundation/logic/view/PopUpMessage.java (hunk 5 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/PopUpMessage.java]: solved.

File: src/main/java/com/fundation/logic/view/loadSaveCriteria/LoadSavePanel.java (hunk 6 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/loadSaveCriteria/LoadSavePanel.java]: Solved.

File: src/main/java/com/fundation/logic/view/HeaderPanel.java (hunk 7 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/HeaderPanel.java]: Solved.

File: src/main/java/com/fundation/logic/view/customElements/CustomJFrame.java (hunk 8 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/customElements/CustomJFrame.java]: Solved.

File: src/main/java/com/fundation/logic/view/customElements/CustomTable.java (hunk 9 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/customElements/CustomTable.java:34]: Solved.

File: src/main/java/com/fundation/logic/view/FooterPanel.java (hunk 2 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/FooterPanel.java:22]: Solved.

File: src/main/java/com/fundation/logic/view/PopUpMessage.java (hunk 4 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/PopUpMessage.java:10]: Solved.

**Pull request merged**